app/_src/kubernetes-ingress-controller/guides/services/tls.md

```diff
@@ -37,15 +38,15 @@ The Ingress API supports TLS termination using the `.spec.tls` field. To termina
 1. Create a `Gateway` resource.
 
     ```yaml
-    apiVersion: gateway.networking.k8s.io/{{ gwapi_version }}
```

Comment:
Why are we changing this to a hardcoded value? Was the version incorrect previously? ( I don't think so but in case that was the case can't we tweak the conditions above to set the correct `gwapi_version`?)

Reply:
This is my mistake; I forgot that this document can switch between different KIC versions. It has been corrected. Thank you!